bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/handler/DanfossAirUnitHandler.java

```diff
@@ -101,12 +109,18 @@ public void initialize() {
             var localCommunicationController = new DanfossAirUnitCommunicationController(
                     InetAddress.getByName(config.host), TCP_PORT);
             this.communicationController = localCommunicationController;
-            this.airUnit = new DanfossAirUnit(localCommunicationController);
+            TimeZoneProvider timeZoneProvider = config.timeZone.isBlank() ? this.timeZoneProvider
+                    : FixedTimeZoneProvider.of(ZoneId.of(config.timeZone));
+
+            airUnit = new DanfossAirUnit(localCommunicationController, timeZoneProvider);
             startPolling();
         } catch (UnknownHostException e) {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR,
                     "@text/offline.communication-error.unknown-host [\"" + config.host + "\"]");
             return;
+        } catch (DateTimeException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR, e.getMessage());
+            return;
```

Comment:
Nitpit: I guess the returns are superfluous here (also in the catch block above).

Reply:
Right, I noticed as well, but decided at the time to stay consistent with the existing catch block. I don't know if it was a left-over or intentionally put there in case more logic was to be added below. Anyway, I agree they are superfluous so I have removed both of them.